README.md

```diff
@@ -84,6 +84,50 @@ batch.Close()
 conn.Close()
 ```
 
+```go
+// to create topics
+topic := "my-topic"
+partition := 0
+
+conn, err := kafka.DialLeader(context.Background(), "tcp", "localhost:9092", topic, partition)
```

Comment:
For a new topic, will this call succeed or will it fail with something like `UnknownTopicOrPartition`?

Reply:
I ran the code and calling DialLeader with an uncreated topic does not appear to yield an error